src/main/features/directions-questionnaire/views/hearing-exceptional-circumstances.njk

```diff
@@ -0,0 +1,46 @@
+{% extends "layout.njk" %}
+{% from "form.njk" import csrfProtection, radioOption, textArea, saveAndContinueButton, errorSummary %}
+
+
+{% if party === MadeBy.CLAIMANT %}
+  {% set courtName = 'Placeholder court name' %}
```

Comment:
This will hold the name of the court that the defendant chose, which currently we can't do until we save the DQ model as part of the defendant response journey (We currently have a story to fill in the gaps in the claimant response side of things that depend on this ROC-5409). 

If the court finder doesn't return a court properly it will trigger the fallback page (the logic for the fallback page is here, and it works functionally but the story containing the content for is still waiting to go into development (ROC-5342)